### ISSUE TITLE
Integrate Javed's tool for visualization and documentation of all ontology fragments  

### ISSUE THREAD

**Issue by rjyounes** [ld4l-labs/ontology#45]
We need to get the source code for Javed's tool so that we can regenerate the documentation as we make modifications to the repository, rather than asking him to regenerate the files. I think the files are already out of sync with version 1.0.1.

@zimeon Do you want to take this on, or make recommendations about how this would best be accomplished?

**Comment by zimeon** [comment]
Please ask @mjaved495 to rerun it or to help you automate/run it, but I also think this should not take priority over the bib2lod work

**Comment by mjaved495** [comment]
I will make the code available on github in this week. Sorry for the delay.

**Comment by rjyounes** [comment]
@zimeon Don't worry, I am not going to work on this myself now, which is why I brought it to your attention.

**Comment by dazza-codes** [comment]
Consider https://github.com/pgfearo/xmlspectrum

**Comment by rjyounes** [comment]
@mjaved495 Can you get this done by the end of next week so I can prepare it for the release?

**Comment by mjaved495** [comment]
Can I assume that ontology files are up to date and no more changes will be made to the ontology files ?

**Comment by rjyounes** [comment]
No. I thought you were going to put the source code in github so that we could fork it into ld4l-labs and regenerate it as a pre-release task.

**Comment by mjaved495** [comment]
I did it already. Here is the link. 
https://github.com/mjaved495/LD4LOntologyDocBuilder
you should use "ontologyfilesreader" package to regenerate.
